lib/compiler/test/compile_SUITE.erl

```diff
@@ -935,6 +935,23 @@ do_warnings_2([], Next, F) ->
     do_warnings_1(Next, F).
 
 
+%% Test that ERL_COMPILER_OPTIONS are correctly retrieved
+%% by env_compiler_options/0
+
+env_compiler_options(_Config) ->
+    Cases = [
+        {"bin_opt_info", [bin_opt_info]},
+        {"'S'", ['S']},
+        {"{source, \"test.erl\"}", [{source, "test.erl"}]},
+        {"[{d,macro_one,1},{d,macro_two}]", [{d, macro_one, 1}, {d, macro_two}]},
+        {"[warn_export_all, warn_export_vars]", [warn_export_all, warn_export_vars]}
+    ],
+    F = fun({Env, Expected}) ->
+        ?line true = os:putenv("ERL_COMPILER_OPTIONS", Env),
```

Comment:
Don't use the ?line macros. All uses of ?line macros have been removed from most test suites in OTP.